content/en/docs/tutorials/pod-security/ns-level-pss.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Applying Pod Security Standards at Namespace level
```

Comment:
Are there any major differences between NS level and cluster level? If not, we can simplify this and add the content to the other tutorial. We can say you can apply at cluster or namespace level (and in the relevant commands, tell the user what they'd change to apply to a namespace). If the steps are too different we can even have two sections in that page: Applying at the cluster level and applying at the NS level. 

Let me know what you think.

Reply:
Yes IMO, both are pretty different and can be done without following the other. IIRC, except installing KinD and kubectl, most steps are different, but the final outcome might look same. 

Perhaps I can modify the tutorial for NS level and cluster level to show how PSS is applied to all namespaces and for namespace level it can applied one by one. Maybe that will make the difference clearer since it seems the outcome of both the tutorials is identical as they are written today!!

Reply:
That's a good idea, following a common use case for each would be nice. You could explain why you'd apply for one NS at a time and show them how.

Reply:
done.